xarray/plot/facetgrid.py

```diff
@@ -112,7 +112,7 @@ class FacetGrid(Generic[T_Xarray]):
     data: T_Xarray
     name_dicts: np.ndarray
     fig: Figure
-    axes: np.ndarray
+    axs: np.ndarray
```

Comment:
I'm sure there's annoyance with not being consistent with very popular upstream terminology as well. 

Because both the terms are very popular. See the example:
https://docs.xarray.dev/en/stable/user-guide/plotting.html#facetgrid-objects

And then check this example:
https://docs.xarray.dev/en/stable/user-guide/plotting.html#maps

It's only deprecated anyway so there will be plenty of time to remove a single letter.

Reply:
our tests emit quite a few of the deprecation warnings ([log](https://github.com/pydata/xarray/actions/runs/3315225559/jobs/5475539484)). I think they should either be explicitly silenced, or the tests should be changed to use the new spelling.